src/components/OptionList/components/Option/Option.scss

```diff
@@ -164,16 +164,7 @@ $selected-option-identifier-width: rem(3px);
     }
 
     &.select::before {
-      background: var(--p-interactive);
-      content: '';
-      position: absolute;
-      top: 0;
-      left: -1 * spacing(tight);
-      height: 100%;
-      display: block;
-      width: $selected-option-identifier-width;
```

Comment:
You no longer use this variable within the file, and therefor it should be removed